platform/barefoot/docker-syncd-bfn/syncd_init_common.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+CMD_SYNCD=/usr/bin/syncd
```

Comment:
not sure what these changes are.

Reply:
This syncd_init_common.sh configures the default async/sync mode for syncd. Since this file is located in sairedis and installed with syncd packages, I copied a file with sync mode enabled here and let it cover the original syncd_init_common.sh file after it is installed with the syncd packages. As such, the sync mode can be enabled by default for both orchagent and syncd. The aim of these changes  is only to test if enabling sync mode can pass the PR checkers.

Reply:
where is this syncd_init_common.sh? can you submit pr to properly change the original file?

Reply:
The file is in sonic-sairedis. I thought we are not going to enable sync mode by default soon, so I made the changes in this PR as a temporary solution only to test sync mode with minimal changes in the submodules. I will create another PR that properly changes the original file when we want to enable sync mode.

Reply:
I am closing this PR since we already confirmed that it could pass all PR checkers with sync mode enabled. Will create another PR that enables sync mode more properly in the future when we want to enable it by default.